Upgrade EUI to v105.0.0

acstll opened the pull request:
`104.1.0` ⏩ `105.0.0`

[Questions? Please see our Kibana upgrade FAQ.](https://github.com/elastic/eui/blob/main/wiki/eui-team-processes/upgrading-kibana.md#faq-for-kibana-teams)

## Package updates

### `@elastic/eui`

#### [`v105.0.0`](https://github.com/elastic/eui/releases/v105.0.0)

- Added marked row styling via the classes `euiDataGridRow--marked` and `euiTableRow--marked` for `EuiDataGrid` and `EuiBasicTable`  ([#8834](https://github.com/elastic/eui/pull/8834))
- Added component tokens: ([#8834](https://github.com/elastic/eui/pull/8834))
  - `dataGridRowBackgroundMarked`
  - `dataGridRowBackgroundMarkedHover`
  - `dataGridRowBorderActive`
  - `dataGridRowBorderHover`
  - `dataGridRowBorderMarked`
  - `tableRowBackgroundMarked`
  - `tableRowBackgroundMarkedHover`
- Added `EuiFlyoutChild` and `EuiFlyoutSessionProvider` ([#8771](https://github.com/elastic/eui/pull/8771))
- Added `setListOptionRefs` prop on `EuiComboBoxList` ([#8829](https://github.com/elastic/eui/pull/8829))

**Breaking changes**

- Removed `iInCircle` icon (use `info` instead) ([#8841](https://github.com/elastic/eui/pull/8841))
- Removed `questionInCircle` icon (use `question` instead) ([#8841](https://github.com/elastic/eui/pull/8841))

**Accessibility**

- Improved the experience of `EuiProgress` by ensuring that determinate updates are read out immediately to screen readers ([#8839](https://github.com/elastic/eui/pull/8839))
- Fixed missing screen reader output for `EuiComboBox` with `options` that have custom `id` attributes ([#8829](https://github.com/elastic/eui/pull/8829))

### `@elastic/eui-theme-borealis`

#### [`v3.2.0`](https://github.com/elastic/eui/blob/main/packages/eui-theme-borealis/changelogs/CHANGELOG_2025.md#v320)

- Added component tokens: ([#8834](https://github.com/elastic/eui/pull/8834))
  - `dataGridRowBackgroundMarked`
  - `dataGridRowBackgroundMarkedHover`
  - `dataGridRowBorderActive`
  - `dataGridRowBorderHover`
  - `dataGridRowBorderMarked`
  - `tableRowBackgroundMarked`
  - `tableRowBackgroundMarkedHover`

## Summary

- **Marked row styling** for `EuiDataGrid` and `EuiBasicTable`: new CSS classes are available for consistent styling of marked rows (different from selected), including hover and cell outline styles (https://github.com/elastic/eui/pull/8834)
- **New flyout component**: `EuiFlyoutChild` allows side-by-side (grouped) flyout panels (https://github.com/elastic/eui/pull/8771)
- **Accessibility fixes:**
  - `EuiComboBox` now supports custom option `id`s (https://github.com/elastic/eui/pull/8829)
  - Updates on `EuiProgress` are properly announced by screen readers (https://github.com/elastic/eui/pull/8839)
- **Icon updates** (breaking change): `iInCircle` and `questionInCircle` icons have been renamed to `info` and `question` respectively (https://github.com/elastic/eui/pull/8841)

acstll committed:
chore: update eui packages

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/316599)
* Commit: 5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/316599#0197e8fa-c105-4ccf-8e55-bcb302ccdad4)

cc @acstll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316599","number":316599,"commit":"5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298"}],"number":316599}
buildkite-pr-comment-->

acstll committed:
chore: update amsterdam package

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/316830)
* Commit: e39d7850a40ae2e37ed90b4f712740831f96879f

### Failed CI Steps
* [Serverless Detection Engine - Exceptions - Security Solution Cypress Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/316830#0197ea57-5d6a-451d-9746-6b16e1cf86ab)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316830#0197ea57-5d6a-451d-9746-6b16e1cf86ab) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316830/jobs/0197ea57-5d6a-451d-9746-6b16e1cf86ab/artifacts/0197ea8b-781a-4173-be5d-ed4ea740e0f5) Serverless Detection Engine - Exceptions - Security Solution Cypress Tests #<span></span>1 / Auto populate exception with Alert data Should create a Rule exception from Alerts take action button and change multiple exception items without resetting to initial auto-prefilled entries Should create a Rule exception from Alerts take action button and change multiple exception items without resetting to initial auto-prefilled entries

### History
* :broken_heart: [Build #316599](https://buildkite.com/elastic/kibana-pull-request/builds/316599) failed 5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298

cc @acstll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316830","number":316830,"commit":"e39d7850a40ae2e37ed90b4f712740831f96879f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316599","number":316599,"commit":"5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298"}],"number":316830}
buildkite-pr-comment-->

acstll committed:
[Security Solution] Update jest test to handle EuiProgress markup

acstll committed:
[UnifiedFieldList] Update jest test after EuiProgress changes

The markup now includes screen reader-only output,
see https://github.com/elastic/eui/pull/8839

acstll committed:
[Security Solution] Update snapshots

acstll committed:
[Security Solution] Force click in Cypress task (not good)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/317735)
* Commit: 8010517d0ea1c6f58ba34e2d7cbe4832bc86f434

### Failed CI Steps
* [Explore - Security Solution Cypress Tests #2](https://buildkite.com/elastic/kibana-pull-request/builds/317735#0197f476-3337-4689-bdc7-9036a127728c)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317735#0197f476-3337-4689-bdc7-9036a127728c) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317735/jobs/0197f476-3337-4689-bdc7-9036a127728c/artifacts/0197f4b7-04ce-434b-9545-d1c98024b13d) Explore - Security Solution Cypress Tests #<span></span>2 / Overview Page Host stats render with correct values Host stats render with correct values
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317735#0197f476-3337-4689-bdc7-9036a127728c) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317735/jobs/0197f476-3337-4689-bdc7-9036a127728c/artifacts/0197f4b7-04cd-47ca-99e8-7a3d1c94b5e1) Explore - Security Solution Cypress Tests #<span></span>2 / Overview Page Network stats render with correct values Network stats render with correct values

### History
* :broken_heart: [Build #316830](https://buildkite.com/elastic/kibana-pull-request/builds/316830) failed e39d7850a40ae2e37ed90b4f712740831f96879f
* :broken_heart: [Build #316599](https://buildkite.com/elastic/kibana-pull-request/builds/316599) failed 5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298

cc @acstll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317735","number":317735,"commit":"8010517d0ea1c6f58ba34e2d7cbe4832bc86f434"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316830","number":316830,"commit":"e39d7850a40ae2e37ed90b4f712740831f96879f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316599","number":316599,"commit":"5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298"}],"number":317735}
buildkite-pr-comment-->

acstll committed:
[Security Solution] Update Cypress test after EuiProgress update

acstll committed:
[Security Solution] Improve comment in Cypress test

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/317942)
* Commit: e0a90cd162d351856f5ec6e5196508921f107476

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/71613a69d5dbb83e69def31f076d7c7796c4528b) | [after](https://github.com/elastic/kibana/commit/e0a90cd162d351856f5ec6e5196508921f107476) | diff |
| --- | --- | --- | --- |
| `kbnUiSharedDeps-npmDll` | 6.4MB | 6.4MB | +18.0KB |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/71613a69d5dbb83e69def31f076d7c7796c4528b) | [after](https://github.com/elastic/kibana/commit/e0a90cd162d351856f5ec6e5196508921f107476) | diff |
| --- | --- | --- | --- |
| `@kbn/test-suites-xpack` | 155 | 156 | +1 |
| `@kbn/unified-field-list` | 18 | 20 | +2 |
| total |  |  | +3 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/71613a69d5dbb83e69def31f076d7c7796c4528b) | [after](https://github.com/elastic/kibana/commit/e0a90cd162d351856f5ec6e5196508921f107476) | diff |
| --- | --- | --- | --- |
| `@kbn/test-suites-xpack` | 169 | 170 | +1 |
| `@kbn/unified-field-list` | 18 | 20 | +2 |
| total |  |  | +3 |


</details>


### History
* :broken_heart: [Build #317735](https://buildkite.com/elastic/kibana-pull-request/builds/317735) failed 8010517d0ea1c6f58ba34e2d7cbe4832bc86f434
* :broken_heart: [Build #316830](https://buildkite.com/elastic/kibana-pull-request/builds/316830) failed e39d7850a40ae2e37ed90b4f712740831f96879f
* :broken_heart: [Build #316599](https://buildkite.com/elastic/kibana-pull-request/builds/316599) failed 5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298

cc @acstll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317942","number":317942,"commit":"e0a90cd162d351856f5ec6e5196508921f107476"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317735","number":317735,"commit":"8010517d0ea1c6f58ba34e2d7cbe4832bc86f434"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316830","number":316830,"commit":"e39d7850a40ae2e37ed90b4f712740831f96879f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316599","number":316599,"commit":"5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298"}],"number":317942}
buildkite-pr-comment-->

elasticmachine committed:
Merge branch 'main' into eui/v105.0.0

acstll commented on src/platform/packages/shared/kbn-unified-field-list/src/components/field_stats/field_stats.test.tsx:
```diff
@@ -434,7 +434,8 @@ describe('UnifiedFieldList FieldStats', () => {
     expect(wrapper.text()).toBe(
-      'Top values"success"41.5%"info"37.1%"security"10.1%"warning"5.0%"error"3.4%"login"2.7%Calculated from 1624 records.'
+      // eslint-disable-next-line prettier/prettier
+      'Top values\"success\"41.5%0.41533251231527096\"info\"37.1%0.3713054187192118\"security\"10.1%0.10129310344827586\"warning\"5.0%0.050492610837438424\"error\"3.4%0.03417487684729064\"login\"2.7%0.027401477832512317Calculated from 1624 records.'
```
> Is there a way to disable extra annotations for the progress bar in our case?

I don't think there's a way to disable those in the markup because it's going to be there in the DOM (via `EuiScreenReaderOnly`) and in the tests if checking for `textContent`, that's why the value is there twice…

but I do think we can make it better and actually helpful if we pass the formatted percentage as `valueText` to `EuiProgress`, because currently the screen-reader-only output is showing that `0.41533251231527096`, we can make it `41.5%`, and it'll still be twice (in `textContent`) but that'd be expected…

I can try and make that change, let me know what you think

acstll replied:
…but that would actually render the value visually on top of the progress bar and that's not what we want 😅

🤔

acstll replied:
I'm thinking one option could be to refactor that section (outlined in green) and replace the current labels (small flex layout) with the built-in `EuiProgress` ones… the one thing that would change is the tooltip, instead of 2 we'd have 1 wrapping `EuiProgress` and not separate for label and value text.

@jughosta let me know what you think 🙂 

@mgadewoll or do you see any possibilities to adjust `EuiProgress`? something like a prop to format sr-only output? also interested in your opinion in general…

<img width="622" height="529" alt="image" src="https://github.com/user-attachments/assets/61d1142e-2bad-41f1-a80a-46e9af4ee6b0" />

https://github.com/elastic/kibana/blob/f368fadd075788d34619f5daea3215034f7c0352/src/platform/packages/shared/kbn-unified-field-list/src/components/field_stats/field_top_values_bucket.tsx#L96-L161

acstll replied:
and another solution, maybe the most efficient imho, would be to:
- refactor the test a bit to do a more granular check instead of just `textContent`
- on the EUI side, evaluate if we should improve this sr-only output to be something different than the raw `value` when not using `valueText`

acstll replied:
We pushed an update that:
- refactors `FieldTopValuesBucketParams` to use built-in labels from `EuiProgress`, making the component a bit simpler
- removes duplicated strings in tests

https://github.com/elastic/kibana/pull/226882/commits/1fbe978ccd6bee8beac2b9516278626bc882bb85

@jughosta please take another look when you get a chance 🙂

jughosta replied:
Thanks! 

I'm not seeing `0.41533251231527096` under `euiScreenReaderOnly` anymore which is great! Although `[object Object] ` sneaked in now. 

<img width="1568" height="1150" alt="Screenshot 2025-07-11 at 18 34 31" src="https://github.com/user-attachments/assets/dc9fcfd3-a73c-41a1-bb57-dba54090b5ea" />

Also, is there a way to avoid double titles (as browser title and eui tooltip)?
![Jul-11-2025 18-35-19](https://github.com/user-attachments/assets/ed6ce795-fc65-4266-8a5c-234488a49d50)

mgadewoll replied:
Imho, the `title` in general doesn't really have much purpose for `EuiProgress` as it simply repeats the text content and for a11y reasons it's not really useful as a) `title` is read very inconsistently and b) we just introduced better screen reader output support.
I'd think we should be fine to move ahead here, `EuiProgress` has titles in other places if used correctly with `label` and `valueText`, it's not a new additional to the component. It only shows up here because the label was composed unexpectedly before. We can remove the `title` on EUI side separately.

acstll replied:
@jughosta thank again! It's expected that `[object Object]` is still there. The fix will be released with the next release this week. We think it's ok to proceed as-is anyway as we already have the fix queued and are only waiting to finish this previous weeks release. Same for the title.

acstll committed:
[UnifiedFieldList] Refactor FieldTopValuesBucket

to use built-in labels from EuiProgress, while keeping the interface intact, including tooltips; tests are also updated

elasticmachine committed:
Merge branch 'main' into eui/v105.0.0

acstll committed:
Remove leftover comment

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/318342)
* Commit: 1b31b3a9c5421e00a094a27365fcfe42702acb74

### Failed CI Steps
* [Defend Workflows Cypress Tests on Serverless #1](https://buildkite.com/elastic/kibana-pull-request/builds/318342#0197fa54-3cbb-46f2-aa0a-9733d7cd1dc7)

### History
* :green_heart: [Build #317942](https://buildkite.com/elastic/kibana-pull-request/builds/317942) succeeded e0a90cd162d351856f5ec6e5196508921f107476
* :broken_heart: [Build #317735](https://buildkite.com/elastic/kibana-pull-request/builds/317735) failed 8010517d0ea1c6f58ba34e2d7cbe4832bc86f434
* :broken_heart: [Build #316830](https://buildkite.com/elastic/kibana-pull-request/builds/316830) failed e39d7850a40ae2e37ed90b4f712740831f96879f
* :broken_heart: [Build #316599](https://buildkite.com/elastic/kibana-pull-request/builds/316599) failed 5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298

cc @acstll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318342","number":318342,"commit":"1b31b3a9c5421e00a094a27365fcfe42702acb74"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317942","number":317942,"commit":"e0a90cd162d351856f5ec6e5196508921f107476"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317735","number":317735,"commit":"8010517d0ea1c6f58ba34e2d7cbe4832bc86f434"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316830","number":316830,"commit":"e39d7850a40ae2e37ed90b4f712740831f96879f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316599","number":316599,"commit":"5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298"}],"number":318342}
buildkite-pr-comment-->

acstll committed:
[UnifiedFieldList] Remove title attr from label in FieldTopValuesBucket

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/318363)
* Commit: 405545bcf39fec52c4f362e1c32306efef42c9be

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/436db09abdbdbcdd33868827f9cf32f248b13068) | [after](https://github.com/elastic/kibana/commit/405545bcf39fec52c4f362e1c32306efef42c9be) | diff |
| --- | --- | --- | --- |
| `aiops` | 504.3KB | 504.2KB | -95.0B |
| `apm` | 2.6MB | 2.6MB | -95.0B |
| `discover` | 1.1MB | 1.1MB | -95.0B |
| `lens` | 1.5MB | 1.5MB | -95.0B |
| `ml` | 5.4MB | 5.4MB | -95.0B |
| `securitySolution` | 9.8MB | 9.8MB | -95.0B |
| `slo` | 977.5KB | 977.5KB | -95.0B |
| `transform` | 621.8KB | 621.7KB | -95.0B |
| total |  |  | -760.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/436db09abdbdbcdd33868827f9cf32f248b13068) | [after](https://github.com/elastic/kibana/commit/405545bcf39fec52c4f362e1c32306efef42c9be) | diff |
| --- | --- | --- | --- |
| `kbnUiSharedDeps-npmDll` | 6.4MB | 6.4MB | +18.0KB |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/436db09abdbdbcdd33868827f9cf32f248b13068) | [after](https://github.com/elastic/kibana/commit/405545bcf39fec52c4f362e1c32306efef42c9be) | diff |
| --- | --- | --- | --- |
| `@kbn/test-suites-xpack` | 151 | 152 | +1 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/436db09abdbdbcdd33868827f9cf32f248b13068) | [after](https://github.com/elastic/kibana/commit/405545bcf39fec52c4f362e1c32306efef42c9be) | diff |
| --- | --- | --- | --- |
| `@kbn/test-suites-xpack` | 163 | 164 | +1 |


</details>


### History
* :broken_heart: [Build #318342](https://buildkite.com/elastic/kibana-pull-request/builds/318342) failed 1b31b3a9c5421e00a094a27365fcfe42702acb74
* :green_heart: [Build #317942](https://buildkite.com/elastic/kibana-pull-request/builds/317942) succeeded e0a90cd162d351856f5ec6e5196508921f107476
* :broken_heart: [Build #317735](https://buildkite.com/elastic/kibana-pull-request/builds/317735) failed 8010517d0ea1c6f58ba34e2d7cbe4832bc86f434
* :broken_heart: [Build #316830](https://buildkite.com/elastic/kibana-pull-request/builds/316830) failed e39d7850a40ae2e37ed90b4f712740831f96879f
* :broken_heart: [Build #316599](https://buildkite.com/elastic/kibana-pull-request/builds/316599) failed 5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298

cc @acstll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318363","number":318363,"commit":"405545bcf39fec52c4f362e1c32306efef42c9be"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318342","number":318342,"commit":"1b31b3a9c5421e00a094a27365fcfe42702acb74"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317942","number":317942,"commit":"e0a90cd162d351856f5ec6e5196508921f107476"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317735","number":317735,"commit":"8010517d0ea1c6f58ba34e2d7cbe4832bc86f434"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316830","number":316830,"commit":"e39d7850a40ae2e37ed90b4f712740831f96879f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316599","number":316599,"commit":"5d2430ee5ca8aaf6ad5e8cb14cf58243285ec298"}],"number":318363}
buildkite-pr-comment-->

elasticmachine committed:
Merge branch 'main' into eui/v105.0.0

jughosta commented on src/platform/packages/shared/kbn-unified-field-list/src/components/field_stats/field_top_values_bucket.tsx:
```diff
@@ -99,64 +99,63 @@ const FieldTopValuesBucket: React.FC<FieldTopValuesBucketProps> = ({
+          }
+          labelProps={{
+            title: undefined,
+          }}
```
It used to wrap the text inside the label. Can we keep this behaviour?

Before:
<img width="815" height="523" alt="Screenshot 2025-07-14 at 11 28 21" src="https://github.com/user-attachments/assets/b567038a-79ef-4321-974e-e13b5a3ae35c" />

After:
<img width="792" height="571" alt="Screenshot 2025-07-14 at 11 27 27" src="https://github.com/user-attachments/assets/c14108e9-5a93-4c10-80f6-19de71a3786a" />

acstll replied:
yes! we'll take care of it 🙏

mgadewoll replied:
@jughosta It's updated and should properly break to max 3 lines as before now.

<img width="399" height="607" alt="Screenshot 2025-07-14 at 12 12 36" src="https://github.com/user-attachments/assets/a9479e5b-e220-43ac-86fe-a2f0900e4177" />

jughosta commented on package.json:
```diff
@@ -127,9 +127,9 @@
-    "@elastic/eui-theme-borealis": "3.1.0",
+    "@elastic/eui": "105.0.0",
+    "@elastic/eui-amsterdam": "npm:@elastic/eui@105.0.0-amsterdam.0",
+    "@elastic/eui-theme-borealis": "3.2.0",
```
A Summary column icon looks broken now. Can you please update it in `src/platform/packages/shared/kbn-unified-data-table/src/components/data_table_summary_column_header.tsx:57`?

<img width="566" height="155" alt="Screenshot 2025-07-14 at 11 22 11" src="https://github.com/user-attachments/assets/05d04182-1613-430d-8b7f-a86aec66aeaf" />

mgadewoll committed:
refactor: unset truncated progress label wrapping

mgadewoll committed:
refactor: update renamed icon

mgadewoll committed:
refactor: update renamed icon

rmyz reviewed (approved):
infra & services changes LGTM

jughosta reviewed (approved):
Thanks! LGTM 👍

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/318664)
* Commit: a569ad5d792dd357379a34f5600cb7353b381070

### Failed CI Steps
* [FTR Configs #116](https://buildkite.com/elastic/kibana-pull-request/builds/318664#01980892-e7fd-4ee0-a38a-27593f026261)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/318664#01980892-e7fd-4ee0-a38a-27593f026261) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/318664/jobs/01980892-e7fd-4ee0-a38a-27593f026261/artifacts/019808bb-f262-4aa3-948d-621ebc3e9a65) FTR Configs #<span></span>116 / serverless search UI Serverless Query Rules Overview Creating a query ruleset from an empty deployment should be able to create a new ruleset

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/2a368e2c69ce8ae52be7a238890c198dc5f80ce1) | [after](https://github.com/elastic/kibana/commit/a569ad5d792dd357379a34f5600cb7353b381070) | diff |
| --- | --- | --- | --- |
| `aiops` | 504.3KB | 504.3KB | -33.0B |
| `apm` | 2.6MB | 2.6MB | -33.0B |
| `cloudSecurityPosture` | 559.0KB | 559.0KB | -8.0B |
| `discover` | 1.1MB | 1.1MB | -41.0B |
| `esqlDataGrid` | 160.0KB | 160.0KB | -8.0B |
| `lens` | 1.5MB | 1.5MB | -33.0B |
| `ml` | 5.4MB | 5.4MB | -33.0B |
| `searchQueryRules` | 121.2KB | 121.2KB | -16.0B |
| `securitySolution` | 9.8MB | 9.8MB | -41.0B |
| `slo` | 977.5KB | 977.5KB | -41.0B |
| `transform` | 621.8KB | 621.7KB | -33.0B |
| `unifiedDocViewer` | 267.6KB | 267.5KB | -8.0B |
| total |  |  | -328.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/2a368e2c69ce8ae52be7a238890c198dc5f80ce1) | [after](https://github.com/elastic/kibana/commit/a569ad5d792dd357379a34f5600cb7353b381070) | diff |
| --- | --- | --- | --- |
| `kbnUiSharedDeps-npmDll` | 6.4MB | 6.4MB | +18.0KB |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/2a368e2c69ce8ae52be7a238890c198dc5f80ce1) | [after](https://github.com/elastic/kibana/commit/a569ad5d792dd357379a34f5600cb7353b381070) | diff |
| --- | --- | --- | --- |
| `@kbn/test-suites-xpack` | 150 | 151 | +1 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/2a368e2c69ce8ae52be7a238890c198dc5f80ce1) | [after](https://github.com/elastic/kibana/commit/a569ad5d792dd357379a34f5600cb7353b381070) | diff |
| --- | --- | --- | --- |
| `@kbn/test-suites-xpack` | 162 | 163 | +1 |


</details>


### History
* :green_heart: [Build #318574](https://buildkite.com/elastic/kibana-pull-request/builds/318574) succeeded 09c056d81c4762626716a26e12776e2f66483433
* :green_heart: [Build #318363](https://buildkite.com/elastic/kibana-pull-request/builds/318363) succeeded 405545bcf39fec52c4f362e1c32306efef42c9be
* :broken_heart: [Build #318342](https://buildkite.com/elastic/kibana-pull-request/builds/318342) failed 1b31b3a9c5421e00a094a27365fcfe42702acb74
* :green_heart: [Build #317942](https://buildkite.com/elastic/kibana-pull-request/builds/317942) succeeded e0a90cd162d351856f5ec6e5196508921f107476
* :broken_heart: [Build #317735](https://buildkite.com/elastic/kibana-pull-request/builds/317735) failed 8010517d0ea1c6f58ba34e2d7cbe4832bc86f434
* :broken_heart: [Build #316830](https://buildkite.com/elastic/kibana-pull-request/builds/316830) failed e39d7850a40ae2e37ed90b4f712740831f96879f

cc @acstll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318664","number":318664,"commit":"a569ad5d792dd357379a34f5600cb7353b381070"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318574","number":318574,"commit":"09c056d81c4762626716a26e12776e2f66483433"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318363","number":318363,"commit":"405545bcf39fec52c4f362e1c32306efef42c9be"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318342","number":318342,"commit":"1b31b3a9c5421e00a094a27365fcfe42702acb74"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317942","number":317942,"commit":"e0a90cd162d351856f5ec6e5196508921f107476"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317735","number":317735,"commit":"8010517d0ea1c6f58ba34e2d7cbe4832bc86f434"}],"number":318664}
buildkite-pr-comment-->
